src/wallet/KintoWallet.sol

```diff
@@ -430,12 +444,12 @@ contract KintoWallet is Initializable, BaseAccount, TokenCallbackHandler, IKinto
             if (newSigners[i] == address(0)) revert InvalidSigner();
         }
 
+        // set new owners
         owners = newSigners;
 
-        // change policy if needed
-        if (_policy != SINGLE_SIGNER && newSigners.length == 1) revert InvalidSingleSignerPolicy();
-        if (_policy != signerPolicy) {
-            _setSignerPolicy(_policy);
+        // change policy, if needed
+        if (newPolicy != signerPolicy) {
```

Comment:
lets not let them change to single signer policy for security

Reply:
Sure.

Reply:
Addressed